src/web/browser/ophan/ophan.ts

```diff
@@ -1,4 +1,4 @@
-import {
+import type {
```

Comment:
This was causing warnings, as it is `export type` in guardian/types, importing as a `type` to removes the warnings.